llvm/include/llvm/IR/Instructions.h

```diff
@@ -1836,6 +1837,16 @@ class CallInst : public CallBase {
   bool canReturnTwice() const { return hasFnAttr(Attribute::ReturnsTwice); }
   void setCanReturnTwice() { addFnAttr(Attribute::ReturnsTwice); }
 
+  bool isNonContinuableTrap() const {
```

Comment:
Needs documentation comment. Also wouldn't any "trap" be "noncontinuable"? Can this just be isTrapIntrinsic?

Reply:
[`debugtrap`](https://llvm.org/docs/LangRef.html#llvm-debugtrap-intrinsic) is not `noreturn`; it acts like a breakpoint and implies continuation.